docs/website/content/faq/_index.md

```diff
@@ -12,9 +12,9 @@ Sysl can create diagrams, data schemas, code (Java, JS) and data transformation.
 The set of outputs is open-ended and allows for your own extensions.
 
 ## I found a bug! What do I do?
-To report a bug, please use our [Issues page on GitHub](https://github.com/anz-bank/sysl/issues/new).
+To report a bug, please use our [Issues page on GitHub](https://github.com/anz-bank/sysl/issues/new/choose).
 
-Or, to be a hero, you can fork Sysl, fix the bug yourself and create a pull request to fix it for everyone.
+Or, to be a hero, you can fork Sysl, fix the bug yourself and create a pull request to fix it for everyone. Please have a look at the [contributing guide](CONTRIBUTING.md) for more information.
```

Comment:
is the link to the contributing guide correct? Couldn't find the markdown in the ` docs/website/content/faq/` folder.